packages/fxa-auth-server/lib/payments/subscription-reminders.ts

```diff
@@ -0,0 +1,216 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Logger } from 'mozlog';
+import Stripe from 'stripe';
+import { Container } from 'typedi';
+
+import { ConfigType } from '../../config';
+import { reportSentryError } from '../sentry';
+import { SentEmailParams, Plan } from 'fxa-shared/subscriptions/types';
+import { StripeHelper, TimeSpanInS } from './stripe';
+import { SentEmail } from 'fxa-shared/db/models/auth';
+
+import {
+  DEFAULT_PLAN_LENGTH,
+  DEFAULT_REMINDER_LENGTH,
+} from '../../scripts/subscription-reminders';
```

Comment:
I think it's safe to leave out the default values and require the plan length and reminder length passed into the constructor.  TypeScript would complain if they are missing.

Reply:
I had added this for a test to assert that if no values are passed into `new SubscriptionReminders(...)`, we'd still fall back to the defaults, but it makes sense to just fall back in the CLI and require the values in the constructor.